bridges/pangolin-ropsten/bridge/src/command/handler/parcel.rs

```diff
@@ -16,10 +18,14 @@ pub async fn handle_parcel(opts: ParcelOpts, basic: BasicOptions) -> color_eyre:
         bridge_config.shadow,
         bridge_config.ethereum,
         bridge_config.web3,
+        BridgeName::PangolinRopsten,
     )?;
 
+    // Darwinia client
+    let client = PangolinClientComponent::component(bridge_config.darwinia).await?;
+    let mmr_root = client.get_mmr_root(block).await?;
```

Comment:
I was wrong here that it should be the root of ethereum mmr. 

We need to revert the changes to calculate that for nodes.

Reply:
Fixed https://github.com/darwinia-network/bridger/pull/429/commits/cbe5ff5f639c6af7d6c4acf39afd17723bda1e8d